conans/client/conan_command_output.py

```diff
@@ -231,7 +233,8 @@ def info_graph(self, graph_filename, deps_graph, cwd, template):
 
         template_folder = os.path.dirname(template.filename)
         save(graph_filename,
-             template.render(graph=graph, assets=assets, base_template_path=template_folder))
+             template.render(graph=graph, assets=assets, base_template_path=template_folder,
+                             version=client_version, year=datetime.today().year))
```

Comment:
Don't need to pass the year dynamically, the html can put the year directly like ``<script>document.write(new Date().getFullYear())</script>``

Reply:
Yeah, I just wanted to avoid JS, but seems be more reasonable.

Reply:
Done.